### DISCUSSION TITLE
Out of memory Error

### DISCUSSION THREAD

**Discussion by TheDataCraft** [apache/airflow#27980] [Q&A]
Hello I'm using airflow task.expand() to create about 100 number of a task to run concurrently. 
But though my server has alot of ram  remained i get sigkill ( out of memory ) error for about 80 of them and only 20 remain running !

Is there any configuration i can set to tell airflow use more ram?

I'm using 2.4.3 dockrized airflow.

**Comment by potiuk** [comment]
If you are on Mac (you forgot to explain your setup) then your docker virtual machine in docker desktop is limited by default to 2GB and you can change it by docker desktop configuration (you can Google it)

But other than that this is not Airflw setting to use specific amount of memory. You need to dig deeper in your deployment.

BTW. If you seek further help, it enormously help those  who try to help you in their free time to describe your deployment and configuration that you have - doing a little homework on trying to find out and documenting your installation - even if it takes a little of your precious time, save also a lot of the precious time of others who try to help efficiently.

**Reply by TheDataCraft** [reply]
Hi thanks so much ...

Sorry it was my first time to ask for help and I'm not realy used to it ...

I hosted docker on a linux server with about 64Gb of ram and 16 cpus ...

There is no limit set by docker or me ...

The only configurations i change was the max_run and max_active to have more than 16 tasks running.

It's been a week since I've been searching google and reading airflow documents but found nothing ...

**Reply by potiuk** [reply]
We can't produce anything out of thin air. If you have not found yourself and will not provide us any more data, logs observations, no-one will be able to do anything about it . You need to see where the oom errors is coming from - and whether it is oom at all. You have not shown any evidences/logs that you base your assesment on.
